server/db/seed.js

```diff
@@ -0,0 +1,179 @@
+import faker from 'faker';
+import campaignsService from '../db/services/campaigns';
+import contactListsService from '../db/services/contact_lists';
+import contactsService from '../db/services/contacts';
+import questionsService from '../db/services/questions';
+import scriptsService from '../db/services/scripts';
+import User from './models/users';
+
+function getRandomFourDigitInt() {
+  const floor = Math.ceil(1000);
```

Comment:
Math.ceil(1000) will always return 1000 since it is the smallest integer >= the given argument.
Same goes for Math.floor(10000).
I think you can just rewrite this as `return Math.floor(Math.random() * (10000 - 1000) ) + 1000;`